packages/create-app/templates/default-app/packages/backend/src/plugins/scaffolder.ts

```diff
@@ -26,10 +27,12 @@ export default async function createPlugin({
 
   const cookiecutterTemplater = new CookieCutter({ containerRunner });
   const craTemplater = new CreateReactAppTemplater({ containerRunner });
+  const railsTemplater = new Rails({ containerRunner });
   const templaters = new Templaters();
 
   templaters.register('cookiecutter', cookiecutterTemplater);
   templaters.register('cra', craTemplater);
+  templaters.register('rails', railsTemplater);
```

Comment:
There's really no need to go via the templaters anymore here actually, you could build this implementation straight into the action instead of you want. We'll be migrating away from this as we remove support for the alpha templates.